docs/marine-book/marine-runtime/configuration-file.md

```diff
@@ -76,7 +76,7 @@ Note, that practically the current limit of Wasm memory is limited to 4 GiB, so
 
 Wasm modules specify their initial memory size. If a limit is lower than the sum of initial memory sizes, the service will fail to load into runtime. Setting less than `2 MiB` per module will likely cause this kind of error.
```

Comment:
i'm a little confused by that sentence. i think i know what you are truing to convey but i'm note sure. is this the intent?
```suggestion
Each Wasm module needs to have a declared memory limit. If the limit specified is below 2MiB or exceeds the total memory limit for the service, the service will not load.
```

Reply:
Nonono. There is a single per-service limit. I wanted to tell convey this :

Each module allocates some memory on startup. Usually it is `>1 MiB`. So, on startup service consumes at least ` `modules_number * 1 MiB` memory. To avoid startup fail, specify at least `modules_number * 2 MiB` total memory limit.

Reply:
gotcha. i knew i was off ... one more question: is it only the summed module total that matters or is there a per module floor? i.e., for modules m1, m2, m3, is {(m1, 1Mib), (m2, 1Mib), (m3, 4Mib)} going to work?

Reply:
Only sum works. I can completely remove this section. The goal is to address  the question "i set memory limit to 10KB and cannot load modules, 100KB does the same, what do i do wrong?"

Reply:
I will need to add this to CLI documentation as well. Didin't actually know the fact that 2 MB is the minimum. I can validate that on CLI side

Reply:
It is not true that it is the minimum. It is just a handy value that just exceeds the actual minimum in most cases. It is an advice, not a requirement.

Reply:
I made that a requirement in CLI. Seems reasonable still since for most users this is something they would want - to have less unexpected problems